web/modules/custom/dpl_identity/src/Form/IdentityConfigForm.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Drupal\dpl_identity\Form;
+
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Defines a form for setting identity configuration.
+ */
+class IdentityConfigForm extends FormBase {
+
+  /**
+   * The config factory service.
+   *
+   * @var \Drupal\Core\Config\ConfigFactoryInterface
+   */
+  protected $configFactory;
+
+  /**
+   * Constructs a new IdentityConfigForm object.
+   *
+   * @param \Drupal\Core\Config\ConfigFactoryInterface $config_factory
+   *   The config factory service.
+   */
+  public function __construct(ConfigFactoryInterface $config_factory) {
+    $this->configFactory = $config_factory;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container): IdentityConfigForm {
+    return new static(
+      $container->get('config.factory')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'dpl_identity_config_form';
+  }
+
+  /**
+   * Form submission handler.
+   *
+   * @param array $form
+   *   An associative array containing the structure of the form.
+   * @param \Drupal\Core\Form\FormStateInterface $form_state
+   *   The current state of the form.
+   */
+  public function submitForm(array &$form, FormStateInterface $form_state): void {
+    $values = $form_state->getValues();
+    $this->configFactory->getEditable('dpl_identity.settings')
+      ->set('identity_color', $values['identity_color'])
+      ->save();
+  }
+
+  /**
+   * Builds the form.
+   *
+   * @param array $form
+   *   An associative array containing the structure of the form.
+   * @param \Drupal\Core\Form\FormStateInterface $form_state
+   *   The current state of the form.
+   *
+   * @return array
+   *   The form structure.
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    // Load existing configuration for the form.
+    $config = $this->configFactory->get('dpl_identity.settings');
+
+    $form['identity_color'] = [
+      '#type' => 'color',
+      '#title' => $this->t('Identity Color'),
+      '#default_value' => $config->get('identity_color') ?? '#365342',
+      '#description' => $this->t('Choose library identity color.'),
```

Comment:
Please [add a readable version of the module name as a context to translated strings](https://github.com/danskernesdigitalebibliotek/dpl-cms/blob/45fb37c42adf3e77157033b44509d100d7d83e0b/web/modules/custom/dpl_url_proxy/src/Form/ProxyUrlConfigurationForm.php#L135-L135). 

This makes it easier for translators to know the original of the texts they are working on.